ci/tests/arakoon/extend_promote_demote_test.py

```diff
@@ -122,108 +143,74 @@ def teardown():
             cmd = 'rm -rf {0}'.format(location)
             general.execute_command_on_node(ip, cmd)
```

Comment:
No need to do an execute_command_on_node
you already have an sshclient available at this point, so you can do client.file_delete([files])